spec/support/factory_girl.rb

```diff
@@ -23,7 +23,7 @@
 
   factory :meetup do
     description "Gon' hoop"
-    start_time Time.zone.now
+    start_time Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.